idea_town/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -1,12 +1,91 @@
 export default `
-  <section class="page" data-hook="experiment-page">
-    <h1>{{title}}</h1>
-    {{#isInstalled}}
-      <button data-hook="uninstall">Uninstall</button>
-    {{/isInstalled}}
-    {{^isInstalled}}
-      <button data-hook="install">Install</button>
-    {{/isInstalled}}
+  <section id="details" class="page" data-hook="experiment-page">
+    <div id="details">
+      <div class="details-sticker"></div>
+
+      <div class="details-header-wrapper">
+        <div class="details-header">
+          <h1>{{title}}</h1>
+          <div class="idea-controls">
+            {{#isInstalled}}
+              <button data-hook="install" class="button primary">Enable {{title}}</button>
+            {{/isInstalled}}
+            {{^isInstalled}}
+              <button data-hook="uninstall" class="button primary">Disable {{title}}</button>
+            {{/isInstalled}}
+            <div class="user-count">7,654,321</div>
+          </div>
+        </div>
+      </div>
+
+      <div class="details-content">
+        <div class="details-overview">
+          <img src="{{thumbnail}}" width="260" height="auto">
```

Comment:
You are [indeed](https://html.spec.whatwg.org/multipage/embedded-content.html#attr-dim-height) [correct](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-height): the `height` attribute should be specified as a non-negative integer.

Since it's just getting discarded anyway by the browser, I'll just remove those height="auto" attributes.

BTW, here's an interesting question: is your w3c validator the same version as the one [linked](https://validator.whatwg.org/) by the whatwg? (Seriously, I can't tell)